lib/lambda_ethereum_consensus/beacon/genesis.ex

```diff
@@ -0,0 +1,138 @@
+defmodule Genesis do
```

Comment:
nit: LambdaEthereumConsensus.Beacon.Genesis ?

Also maybe the genesis naming is a bit confusing if we're doing checkpoint sync which is not technically a "genesis" state

Reply:
What about something like StoreSetup or similar?

Reply:
Used StoreSetup